test/test_init.py

```diff
@@ -48,6 +53,63 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.mark.parametrize("dim", [1, 2, 3])
+@pytest.mark.parametrize("nspecies", [0, 1, 2, 10])
```

Comment:
Can probably also be cut down.

Reply:
Why not just 0 and 10? Do we gain any additional information from the in-between values?

Reply:
I changed this - but I do perceive that I needed at least 0, 1, 10 when developing this.  0 is just the original test - no scalar components.  1, and 10 species to help me make sure that everything is working when there are (a) any scalar components at all, (b) many scalar components.